### ISSUE TITLE
Multiple accounts

### ISSUE THREAD

**Issue by cbruiners** [Kemmey/Kemmey-TeslaWatch-Public#570]
I have 1 Tesla I own and one I lease, I have access to both accounts, but is there a way to login to more than 1 account?

**Comment by Kemmey** [comment]
No, but why not invite one account as a driver on the other? Then you can access both from one?!
